docs/development.md

```diff
@@ -223,8 +223,7 @@ There are several files used to configure various aspects of the system.
 * `gradle.properties`, `gradle-wrapper.properties`: Contains the Gradle and Gradle wrapper configuration.
 * `package.json`: Contains the front-end third-party dependencies specification, as well as configurations for automated tasks/routines to be run via NPM.
 * `angular.json`: Contains the Angular application configuration.
-* `.travis.yml`: Contains the Travis CI job configuration.
-* `appveyor.yml`: Contains the AppVeyor CI job configuration.
+* `main.yml`: Contains the Github Actions CI job configuration.
```

Comment:
- This is not accurate because the file name is no longer `main.yml`.
- Use the correct capitalization for GitHub brand, i.e. `GitHub` (not `Github`). Applies everywhere else.
- Also, the tool name is GitHub Actions, without CI at the end of it, so it should be mentioned as such. Applies everywhere else.

Reply:
updated - thanks for the catch